bay/docker/hosts.py

```diff
@@ -171,7 +173,40 @@ def build_host_ip(self):
         Returns the internal IP of the host as seen from the containers during
         build, which is the gateway of the default bridge network.
         """
-        return self.client.inspect_network("bridge")['IPAM']['Config'][0]['Gateway']
+        """
+        Determines the "host" IP containers should use.
+
+        Get the Gateway IP from the docker daemon.
```

Comment:
`Gateway IP` could be reworded `Host DNS`.
And add this: `This DNS resolves to the internal IP address used by the host.`